tests/bud.bats

```diff
@@ -7427,3 +7427,18 @@ EOF
   find ${TEST_SCRATCH_DIR}/buildcontext -ls
   expect_output "" "build should not be able to write to build context"
 }
+
+@test "build-with-timestamp-applies-to-oci-archive" {
+  local outpath="${TEST_SCRATCH_DIR}/timestamp-oci.tar"
+
+  run_buildah build -f <(echo 'FROM scratch') --tag=oci-archive:${outpath}.a --timestamp 0
+  run_buildah build -f <(echo 'FROM scratch') --tag=oci-archive:${outpath}.b --timestamp 1
+  sleep 1.1 # sleep at least 1 second, so that timestamps are incremented
+  run_buildah build -f <(echo 'FROM scratch') --tag=oci-archive:${outpath}.c --timestamp 1
+
+  # should be different
+  ! diff "${outpath}.a" "${outpath}.b"
```

Comment:
There is a [warning](https://openscanhub.fedoraproject.org/task/45566/log/added.html) reported by OpenScanHub. Is it a real bug?

Reply:
Thanks, you're quite right. I'd tested the 2nd part (which was the main test) but didn't notice this failed to fail. I've fixed as per one of the suggestions in https://bats-core.readthedocs.io/en/stable/gotchas.html#my-negated-statement-e-g-true-does-not-fail-the-test-even-when-it-should

Reply:
No problem. In the future please check the results from [osh-diff-scan](https://github.com/containers/buildah/pull/6023/checks?check_run_id=39360700780) job submitted through Packit.